website/src/images/companies/powered-soft.svg

```diff
@@ -0,0 +1,28 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1503 1061">
+    <defs>
+        <style>.cls-3{mix-blend-mode:overlay;fill:url(#linear-gradient);}.cls-4{fill:#fff;}</style>
```

Comment:
No colors!

Reply:
Have to have the white fill or there logo does not appear.

Reply:
Fixed it. Next time logos have to go through me. Logos always need to be reworked a bit, so everything looks nice on our website. Reworked 10 logos.